efficientdet/dataloader.py

```diff
@@ -237,6 +240,7 @@ def __call__(self, params):
     anchor_labeler = anchors.AnchorLabeler(input_anchors, params['num_classes'])
     example_decoder = tf_example_decoder.TfExampleDecoder()
 
```

Comment:
What does this tf.autograph.experimental.do_not_convert do?

Reply:
I could debug the code.

Reply:
would it be slower?

Reply:
> would it be slower?

No.